lib/hca/enrollment_system.rb

```diff
@@ -879,9 +879,10 @@ def veteran_to_save_submit_form(
       request = build_form_for_user(current_user, form_id)
 
       veteran['attachments']&.each do |attachment|
-        hca_attachment = HCAAttachment.find_by(guid: attachment['confirmationCode'])
+        guid = attachment['confirmationCode']
+        form_attachment = HCAAttachment.find_by(guid:) || Form1010EzrAttachment.find_by(guid:)
         request['va:form']['va:attachments'] ||= []
```

Comment:
If it can't find an `HCAAttachment`, it'll fall into looking for a `Form1010EzrAttachment`.

Reply:
Should there be a spec/updated spec in the enrollment_system_spec.rb to check for this?

Reply:
I don't think so. There are current attachment tests in the `hca` directory that should cover this, along with the tests I added in this PR.

HCA tests: https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/lib/hca/service_spec.rb#L184

Reply:
I guess my concern was when I remove `|| Form1010EzrAttachment.find_by(guid:)` from this line all of the tests still pass. If this were to be removed nothing would fail but in the actual implementation we would not be getting any of the ezr attachments. Or are you saying that fails another test other than the `enrollment_system_spec.rb`?

Reply:
In order to test the ezr attachment portion, remove that code and run the the following command:
```
rspec spec/lib/form1010_ezr/service_spec.rb:254
```

They are attachment tests for the ezr service and they will fail because `nil` will be returned for `form_attachment = HCAAttachment.find_by(guid:) || Form1010EzrAttachment.find_by(guid:)`

Reply:
I went ahead and removed the feature toggle. I updated the tests to reflect the changes.

Reply:
Okay, I added a new test in `enrollment_system_spec.rb` to confirm EZR attachments are added to the form.